packages/e2e/cypress/e2e/run/create-pipelinerun.cy.js

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2022 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const pipeline = `
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: simple-pipeline
+  namespace: e2e
+spec:
+  tasks:
+    - name: hello
+      taskSpec:
+        steps:
+          - name: echo
+            image: busybox
+            script: |
+              #!/bin/ash
+              echo "Hello World!"
+`;
+describe('Create Simple Pipeline Run', () => {
+  before(() => {
+    cy.exec('kubectl version --client');
+    cy.exec('kubectl create namespace e2e || true');
+    cy.exec(`echo "${pipeline}" | kubectl apply -f -`);
+  });
+  it('should create pipelinerun', function () {
+    cy.visit(
+      `/#/pipelineruns/create?namespace=e2e&pipelineName=simple-pipeline`
+    );
+    cy.contains('button', 'Create').click();
+    cy.contains('simple-pipeline').first().click();
```

Comment:
Ideally the created run would have a unique identifiable name to ensure we're looking at the newly created one and not another run that happens to have the same name, possibly left over from a previous test.

This can be achieved by generating a unique value (e.g. adding a timestamp suffix) and typing it in the 'PipelineRun name' field of the create form.